theano/sandbox/tests/test_rng_mrg.py

```diff
@@ -1189,6 +1190,71 @@ def test_validate_input_types_gpuarray_backend():
         rstate = gpuarray_shared_constructor(rstate)
```

Comment:
This test should be skipped if the new back-end is not available, which is why it is failing on Travis.